src/UID2.Client/UID2Encryption.cs

```diff
@@ -21,12 +21,14 @@ internal static class UID2Encryption
     {
         public const int GCM_AUTHTAG_LENGTH = 16;
         public const int GCM_IV_LENGTH = 12;
+        // Length of a V2 token, the shortest token currently supported
+        public const int MINIMUM_TOKEN_STRING_LENGTH = 180;
```

Comment:
I don't think we need this at class scope, we can move this to the method if it's only used once

Reply:
Removed